app/javascript/components/editor/file-editor/themes.ts

```diff
@@ -0,0 +1,30 @@
+import * as monaco from 'monaco-editor/esm/vs/editor/editor.api'
```

Comment:
@iHiD This is where you can define the custom themes. I've set up two themes (light and dark)